File: Iniciante/1014 - Consumo/Python/1014.py
def main():
    total_distance_traveled = int(input())
    total_fuel_used = float(input())
    average_car_consumption = calculate_average_car_consumption(
        total_distance_traveled, total_fuel_used
    )
    print(f"{average_car_consumption:.3f} km/l")


def calculate_average_car_consumption(distance, fuel):
    return distance / fuel


if __name__ == "__main__":
    main()

File: Iniciante/1014 - Consumo/JavaScript/1014.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const [totalDistanceTraveled, totalFuelUsed] = lines.map(Number);
  const averageCarConsumption = calculateAverageCarConsumption(
    totalDistanceTraveled,
    totalFuelUsed
  );
  console.log(`${averageCarConsumption.toFixed(3)} km/l`);
}

function calculateAverageCarConsumption(distance, fuel) {
  return distance / fuel;
}

main();
